src/tiered/tiered_work.c

```diff
@@ -114,9 +114,9 @@ __tiered_push_new_work(WT_SESSION_IMPL *session, WT_TIERED_WORK_UNIT *entry)
 {
     /*
      * Bump the in use count lock on the dhandle, this is kept until the work unit is freed. This
-     * prevents the an otherwise idle dhandle from being swept and freed.
+     * prevents an otherwise idle dhandle from being swept and freed.
      */
-    (void)__wt_atomic_addi32(&entry->tiered->iface.session_inuse, 1);
+    WT_DHANDLE_ACQUIRE((WT_DATA_HANDLE *)&entry->tiered->iface);
```

Comment:
At the risk of being pedantic, I would also include a comment here saying that it is safe to acquire the dhandle without the handle list read lock because we know the handle can't be swept out from under us because our session has its own reference to the handle at this point. (See `__session_find_shared_dhandle` for the other place we use `WT_DHANDLE_ACQUIRE`.)